capture/crawler_valor.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8 -*-
+from goose import Goose
+import pymongo
+from bs4 import BeautifulSoup
+import requests
+import re
+import pandas as pd
```

Comment:
You don't need to import `re` nor `pandas`. It's a good idea to remove these imports.